casdoorsdk/jwt.go

```diff
@@ -24,6 +24,12 @@ type Claims struct {
 	User
 	AccessToken string `json:"accessToken"`
 	jwt.RegisteredClaims
+	TokenType string `json:"TokenType"`
```

Comment:
Did you test your code? Shouldn't it be `json:"tokenType"`?

Reply:
There is `tokenType` which equals to `access-token` in both refresh and access tokens. But `TokenType` which equals to `refresh-token` presents only in refresh tokens. Not sure why. Maybe it is because of misspell in server code.